<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IDCardDlg_Form</class>
 <widget class="QWidget" name="IDCardDlg_Form">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>身份证读取</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qdilgcamera.qrc">
    <normaloff>:/QDilgCamera/Resources/pic/eloam.ico</normaloff>:/QDilgCamera/Resources/pic/eloam.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(188,221,221);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 85, 0);
font: 75 14pt &quot;Aharoni&quot;;</string>
   </property>
   <property name="text">
    <string>中华人民共和国居民身份证</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>441</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>正面</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>姓名</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>性别</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>民族</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>出生</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>住址</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>公民身份证号码</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="m_IDCardName">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="m_IDCardSex">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="m_IDCardNation">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="m_IDCardAge_Y">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="m_IDCardAdreass">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="m_IDCardIDnum">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>110</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>年</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="m_IDCardAge_M">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>月</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="m_IDCardAge_D">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>197</x>
      <y>100</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>日</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_showImage">
    <property name="geometry">
     <rect>
      <x>283</x>
      <y>20</y>
      <width>121</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 170, 255);
background-color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string>头像</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>441</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>反面</string>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>签发机关</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>有效期限</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="m_m_IDCardIdIssued">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="m_IDCardData">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="m_PbReadCard">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>470</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开始读卡</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_PbReadIdCard">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>470</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保存图片</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_PbReadCardExit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>470</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ShowCardState">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>505</y>
     <width>411</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 85, 127);</string>
   </property>
   <property name="text">
    <string>。。。</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_PbReadIdCard_SaveAs">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>470</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>导出</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="qdilgcamera.qrc"/>
 </resources>
 <connections/>
</ui>
